types.ts

```diff
@@ -58,8 +58,19 @@ export interface Game {
 }
 
 export interface Ruleset {
+  /**
+   *  @description Name of the ruleset being used to run this game. Possible values include: standard, solo, royale, squad, constrictor, wrapped. See {@link https://docs.battlesnake.com/guides/playing/modes Game Modes} for more information on each ruleset
```

Comment:
You can use the `@see` tag here
```suggestion
   *  @description Name of the ruleset being used to run this game. Possible values include: standard, solo, royale, squad, constrictor, wrapped. 
   * @see {@link https://docs.battlesnake.com/guides/playing/modes Game Modes} for more information on each ruleset
```